src/OpenTK.Windowing.Common/Interfaces/IGraphicsContext.cs

```diff
@@ -26,5 +26,13 @@ public interface IGraphicsContext
         /// Makes no GraphicsContext current one on the calling thread.
         /// </summary>
         void MakeNoneCurrent();
+
+        /// <summary>
+        /// Gets or sets the swap interval (the number of screen updates to wait between calling <see cref="SwapBuffers"/>).
+        /// </summary>
+        /// <value>
+        /// The swap interval.
+        /// </value>
+        int SwapInterval { get; set; }
```

Comment:
Actually, I have a slight nitpick.
I would move this to be above all of the function definitions in this file.
So it would be below `IsCurrent` and above `SwapBuffers()`.